.github/ISSUE_TEMPLATE/gc_member_onboarding.md

```diff
@@ -0,0 +1,192 @@
+---
+name: GC Member Onboarding
+about: Onboard a new member of the Governance Committee
+title: 'GC ONBOARDING: <GH_USERNAME>'
+labels: area/onboarding
+assignees: ''
+---
+
+<!-- Please remember to change the title of this issue by replacing
+<GH_USERNAME> with the actual GitHub username of the GC member to be
+onboarded. -->
+
+This issue should be assigned to both an existing member and a newly elected
+member, and the actions should be performed by both in collaboration. This
+ensures fluid communication and a successful onboarding into the
+OpenTelemetry Governance Committee.
+
+The issue should be `Closed as completed` by the new member after all steps
+have been completed.
+
+## Assumptions
+These assumptions ensure that the rest of the guide can be followed
+successfully.
+
+The existing member must:
+
+1. Have the necessary privileges to perform the actions listed below.
+
+The new member must:
+
+1. Be familiar with basic OpenTelemetry concepts
+   1. [Mission, Vision and Values](https://opentelemetry.io/community/mission/)
+   2. [Concepts](https://opentelemetry.io/docs/concepts/)
+   3. [Specification](https://opentelemetry.io/docs/specs/otel/)
+2. Have read the [OpenTelemetry Governance Committee Charter](https://github.com/open-telemetry/community/blob/main/governance-charter.md)
```

Comment:
is "reading it" enough or is there some "read and comply"?

Reply:
Good point, maybe "Have read and agreed to comply with..."?

Reply:
yes!